### ISSUE TITLE
The training results differ from the paper by 33%

### ISSUE THREAD

**Issue by bizheyandebing** [hisfog/SfMNeXt-Impl#36]
I have completely trained your model, and the difference with the model you provided is close to 33%. Could you please help me check my training files.

![image](https://github.com/hisfog/SfMNeXt-Impl/assets/29300743/41d590aa-a96b-470a-8082-81c91a0ee29e)

![image](https://github.com/hisfog/SfMNeXt-Impl/assets/29300743/b4c9301a-b527-46ba-a607-71402624ee9d)
Below is the content of my training file:

--data_path /home/ubuntu/ubuntu_jixie/temp/kitti-raw
--dataset kitti
--model_name res_088
--backbone resnet_lite
--height 192
--width 640
--batch_size 16
--num_epochs 25
--scheduler_step_size 15
--num_layers 50
--num_features 256
--model_dim 32
--patch_size 16
--dim_out 64
--query_nums 64
--min_depth 0.001
--max_depth 80.0
--eval_mono
--post_process

**Comment by XIAN-XIAN-X** [comment]
I'm having the same issue！Did you use the res50 as the depth network's backbone！Maybe you can try to set the --load_weight_floder!

**Comment by bizheyandebing** [comment]
Yes, I use res50 as the backbone network, If I set --load_weight_floder, how should its parameters be set?

**Comment by XIAN-XIAN-X** [comment]
I haven't done the experiment yet, but I saw a picture posted by the person who succeeded in the experiment, which has the&nbsp;--load_weights_folder&nbsp;item loaded, and you can leave the rest unchanged, add the&nbsp;--load_weights_folder&nbsp;&nbsp;item, and then experiment. The pre-trained file should be downloaded from the link provided by the author, and I feel that this pre-trained model is very important! You can try it and look forward to the results of your experiment




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年4月9日(星期二) 下午4:47
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [hisfog/SfMNeXt-Impl] The training results differ from the paper by 33% (Issue #36)














 
Yes, I use res50 as the backbone network, If I set --load_weight_floder, how should its parameters be set?

**Comment by bizheyandebing** [comment]
I think this setting should have no effect, because I found that the relevant code of the train.py file (lines 137-138h) has been logged out.

Moreover, there is no mention of additional pre-training in the paper.

**Comment by hisfog** [comment]
Apologies for the delayed response, For reproducing results on KITTI，please DO NOT use the latest code release. Instead, you can kindly utilize the following version by
```bash
git checkout 6a1e997f97caef8de080bb2873f71cfbad9a8047
```
which is consistent with the implementation of paper SQLdepth, without any additional modifications.
You can refer to https://github.com/hisfog/SfMNeXt-Impl/issues/26#issuecomment-2006322584, Thanks！
